rundeckapp/grails-app/domain/rundeck/ScheduledExecution.groovy

```diff
@@ -257,6 +257,7 @@ class ScheduledExecution extends ExecutionContext {
         map.scheduleEnabled = hasScheduleEnabled()
         map.executionEnabled = hasExecutionEnabled()
         map.nodeFilterEditable = hasNodeFilterEditable()
+        map.excludeFilterUncheck = hasNodesHealthCheckEnabled()
```

Comment:
if the excludeFilterUncheck is default true/false, then it should not be added to the export map.  (also if there is no filterExclude) in that way the XML/yaml serialization for old jobs will not change

Reply:
@miguelantonio you mean I just need to remove it or check first if the exists like:
```if(this.filterExclude){
                map.nodefilters.filterExclude = this.filterExclude

                if(this.excludeFilterUncheck){
                    map.excludeFilterUncheck = true
                }
            }```

Reply:
as long as the excludeFilterUncheck defaults to false, that is ok.  if it should be default true it would be the inverse

Reply:
@gschueler I sent the fix, the default value is false